recipe/meta.yaml

```diff
@@ -15,10 +14,11 @@ source:
     - 0001-Remove-host-prefix-suffix-when-cross-compiling.patch
 
 build:
-  number: 1
+  number: 2
   run_exports:
-    # ABI seems to be stable between minor versions
-    - {{ pin_subpackage('wxwidgets', max_pin='x.x') }}
+    # ABI seems to be stable
+    # but let's preemptively guard against breakage
```

Comment:
Can you add a link to the issue you raised upstream where they broke things in 3.2.6 and they don't know what they wil ldo in 3.2.7